api_docs/changelog.md

```diff
@@ -19,6 +19,13 @@ clients should check the `zulip_feature_level` field, present in the
 format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
+**Feature level 243**
```

Comment:
You need a blank line before this -- the page doesn't render properly if you don't. Always test your changes -- it's not have to view these docs (/api/changelog).